CHANGELOG.md

```diff
@@ -4,6 +4,58 @@
 `psql` with the `-X` flag to prevent any `.psqlrc` commands from
 accidentally triggering the load of a previous DB version.**
 
+
+## 2.18.0 (2025-01-17)
+
+This release contains performance improvements and bug fixes since
+the 2.17.2 release. We recommend that you upgrade at the next
+available opportunity.
+
+
+**Features**
+* #6901: Add hypertable support for transition tables.
+* #7104: Hypercore table access method.
+* #7271: Push down `order by` in real-time continuous aggregate queries.
+* #7295: Support `alter table set access method` on hypertable.
+* #7341: Vectorized aggregation with grouping by one fixed-size by-value compressed column (such as arithmetic types).
+* #7390: Disable custom `hashagg` planner code.
+* #7411: Change parameter name to enable hypercore table access method.
+* #7412: Add [GUC](https://www.postgresql.org/docs/current/acronyms.html#:~:text=GUC) for the `hypercore_use_access_method` default.
+* #7413: Add GUC for segmentwise recompression.
+* #7433: Add support for merging chunks.
+* #7436: Add index creation on `order by` columns.
+* #7443: Add hypercore function and view aliases.
+* #7455: Support `drop not null` on compressed hypertables.
+* #7458: Support vecorized aggregation with aggregate `filter` clauses that are also vectorizable.
+* #7482: Optimize recompression of partially compressed chunks.
+* #7486: Prevent building against postgres versions with broken ABI.
+* #7521: Add optional `force` argument to `refresh_continuous_aggregate`.
+* #7528: Transform sorting on `time_bucket` to sorting on time for compressed chunks in some cases.
+* #7565: Add hint when hypertable creation fails.
+* #7587: Add `include_tiered_data` parameter to `add_continuous_aggregate_policy` API.
+
+**Bugfixes**
+* #7378: Remove obsolete job referencing `policy_job_error_retention`.
+* #7409: Update `bgw_job` table when altering procedure.
+* #7410: Fix the `aggregated compressed column not found` error on aggregation query.
+* #7426: Fix `datetime` parsing error in chunk constraint creation.
+* #7432: Verify that the heap tuple is valid before using.
+* #7434: Fix the segfault when internally setting the replica identity for a given chunk.
+* #7488: Emit error for transition table trigger on chunks.
+* #7514: Fix the error: `invalid child of chunk append`.
+* #7517: Fix the performance regression on the `cagg_migrate` procedure.
+* #7527: Restart scheduler on error.
+* #7557: Fix null handling for in-memory tuple filtering.
+* #7566: Improve transaction check in CAgg refresh.
+* #7584: Fix NaN-handling for vectorized aggregation.
+
+**Thanks**
+* @bharrisau for reporting the segfault when creating chunks.
+* @k-rus for suggesting that we add a hint when hypertable creation fails.
+* @pgloader for reporting the issue in an internal background job.
```

Comment:
```suggestion
```

Reply:
Edited CHANGELOG.md with release summary and re-ordering of issues, as you'd mentioned in https://docs.google.com/document/d/1XAT2NXcBeOF0GRVYCXX_Puy0YPcVOlG_OLHfo3kUJZI/edit?tab=t.0